packages/core/src/app/customer/CustomerRegistered.test.tsx

```diff
@@ -0,0 +1,302 @@
+import '@testing-library/jest-dom';
+import {
+    Cart,
+    Checkout,
+    CheckoutSelectors,
+    CheckoutService,
+    createCheckoutService,
+    StoreConfig,
+} from '@bigcommerce/checkout-sdk';
+import faker from '@faker-js/faker';
+import userEvent from '@testing-library/user-event';
+import React, { FunctionComponent } from 'react';
+
+import { AnalyticsProviderMock } from '@bigcommerce/checkout/analytics';
+import { createLocaleContext, LocaleContext, LocaleContextType } from '@bigcommerce/checkout/locale';
+import { CheckoutProvider } from '@bigcommerce/checkout/payment-integration-api';
+import { render, screen } from '@bigcommerce/checkout/test-utils';
+
+import { getCart } from '../cart/carts.mock';
+import { getCheckout } from '../checkout/checkouts.mock';
```

Comment:
Also, I want to align on this, when using mocks, do we use the ones in `core` or `test-mocks`?
cc @animesh1987

Reply:
We should try and use test-mocks.